pkg/storage/image.go

```diff
@@ -616,6 +616,8 @@ func splitDockerDomain(name string) (domain, remainder string) {
 	return
 }
 
+// Resolve names resolves an image name into a storage image ID or a fully-qualified image name (domain/repo/image:tag).
```

Comment:
This shoult start `// ResolveNames resolves` ([docs][1]).

[1]: http://golang.org/doc/articles/godoc_documenting_go_code.html

Reply:
oh yeah

Reply:
Fixed